btax/parameters.py

```diff
@@ -21,15 +21,17 @@
 DEFAULT_INDUSTRY_COLS = json.loads(read_from_egg(os.path.join('param_defaults', 'btax_results_by_industry.json')))
 
 
-def translate_param_names(start_year=2016,**user_mods):
+def translate_param_names(start_year=2017,**user_mods):
     """Takes parameters names from UI and turns them into names used in btax
 
     """
 
     # btax_betr_entity_Switch # If this parameter =True, then u_nc default to corp rate
 
-    defaults = dict(DEFAULTS)
     year = start_year-2015
```

Comment:
At first glance, @jdebacker, the hard-coded `2015` here looks like something we would forget over time.  Maybe it should be a named constant at the top of a module.